config/config.go

```diff
@@ -167,15 +167,16 @@ type tfvarsFileWithTerragruntConfig struct {
 // IncludeConfig represents the configuration settings for a parent Terragrunt configuration file that you can
 // "include" in a child Terragrunt configuration file
 type IncludeConfig struct {
-	Source    string `hcl:"source"`
-	Path      string `hcl:"path"`
-	IncludeBy *IncludeConfig
+	Source      string `hcl:"source"`
+	Path        string `hcl:"path"`
+	isIncludeBy *IncludeConfig
```

Comment:
nitpick: `isIncludedBy`

Reply:
I added several tests and renamed isIncludeBy to isIncludedBy.